core/src/main/java/jenkins/fingerprints/FileFingerprintCleanupThread.java

```diff
@@ -50,30 +48,14 @@
  *
  * @author Kohsuke Kawaguchi
  */
-@Extension @Symbol("fingerprintCleanup")
+@Extension(ordinal=-100)
 @Restricted(NoExternalUse.class)
-public class FingerprintCleanupThread extends AsyncPeriodicWork {
+public class FileFingerprintCleanupThread extends FingerprintCleanupThread {
```

Comment:
It used to be a public APi before accmod was introduced. Should be safe enough, I cannot find or imagine an external usage.